spec/features/user_index_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+require 'capybara/rspec'
+
+RSpec.describe 'User Index Page', type: :feature do
+  before do
+    # Set up any necessary data or visit relevant pages
+  end
+
+  it 'displays the username of all other users' do
+    # Test code goes here
+  end
+
+  it 'displays the profile picture for each user' do
+    # Test code goes here
+  end
+
+  it 'displays the number of posts each user has written' do
+    # Test code goes here
+  end
+
+  it 'redirects to user show page when clicking on a user' do
+    # Test code goes here
+  end
```

Comment:
- [ ] Good to have this user index test file. :+1:  But please from the project requirement, you should write a test for the following :point_down: But now you have no test code in your `it` block to test the following:blush: 
  
  -   Tests that asserts you can see the username of all other users.
  -   Tests that asserts you can see the profile picture for each user.
  -   Tests that asserts you can see the number of posts each user has written.
  -   Tests that redirects to the user's show page when you click on a user.